ui/src/components/NavDrawer.js

```diff
@@ -21,6 +20,37 @@ const useStyles = makeStyles(_theme => ({
     fontWeight: 'bold',
     textDecoration: 'underline',
   },
+  logo: {
+    margin: '8px 8px 0 8px',
+    display: 'flex',
+    padding: '4px 4px 0 4px',
+    alignItems: 'center',
+    flexDirection: 'row',
+    height: '60px',
+    [_theme.breakpoints.down('xs')]: {
+      display: 'none',
+    },
+  },
+  logoImage: {
+    transform: 'scale(0.85)',
+  },
+  sectionHeader: {
+    padding: '16px',
+    fontSize: '16px',
+    fontFamily: 'Montserrat,Arial,sans-serif',
```

Comment:
Why `Montserrat`? Do we have a house style w.r.t. fonts?

Reply:
I don't believe we do. We do use this font in the wallet as well though.